kolibri/plugins/learn/assets/src/views/ContentCardListViewItem.vue

```diff
@@ -0,0 +1,318 @@
+        icon="infoPrimary"
+        class="footer-icon"
+        size="mini"
+        :color="$themePalette.grey.v_400"
+        :ariaLabel="$tr('viewInformation')"
+        :tooltip="$tr('viewInformation')"
+        @click="$emit('toggleInfoPanel')"
+      />
+    </div>
+  </router-link>
+
+</template>
+
+
+<script>
+
+  import { validateLinkObject, validateContentNodeKind } from 'kolibri.utils.validators';
+  import {
+    LearningActivities,
+    ContentKindsToLearningActivitiesMap,
+  } from 'kolibri.coreVue.vuex.constants';
+  import TextTruncator from 'kolibri.coreVue.components.TextTruncator';
+  import commonLearnStrings from './commonLearnStrings';
+  import CardThumbnail from './ContentCard/CardThumbnail';
+
+  export default {
+    name: 'ContentCardListViewItem',
+    components: {
+      CardThumbnail,
+      TextTruncator,
+    },
+    mixins: [commonLearnStrings],
+    props: {
+      title: {
+        type: String,
+        required: true,
+      },
+      description: {
+        type: String,
+        default: null,
+      },
+      thumbnail: {
+        type: String,
+        default: null,
+      },
+      channelThumbnail: {
+        type: String,
+        default: null,
+      },
+      channelTitle: {
+        type: String,
+        default: null,
+      },
+      kind: {
+        type: String,
+        required: true,
+        validator: validateContentNodeKind,
+      },
+      level: {
+        type: String,
+        default: '2',
+      },
+      category: {
+        type: String,
+        default: 'math',
+      },
+      progress: {
+        type: Number,
+        required: false,
+        default: 0.0,
+        validator(value) {
+          return value >= 0.0 && value <= 1.0;
+        },
+      },
+      link: {
+        type: Object,
+        required: true,
+        validator: validateLinkObject,
+      },
+      isMobile: {
+        type: Boolean,
+        default: false,
+      },
+      contentId: {
+        type: String,
+        default: null,
+      },
+      copiesCount: {
+        type: Number,
+        default: null,
+      },
+    },
+    computed: {
+      maxTitleHeight() {
+        return 66;
+      },
+      maxDescriptionHeight() {
+        return 40;
+      },
+      displayCategoryAndLevelMetadata() {
+        if (this.category && this.level) {
+          return `${this.category} | ${this.level} `;
+        } else if (this.category) {
+          return this.category;
+        } else if (this.level) {
+          return this.level;
+        } else {
+          return null;
+        }
+      },
+      kindToLearningActivity() {
+        let activity = '';
+        if (Object.values(LearningActivities).includes(this.kind)) {
+          activity = this.kind;
+        } else {
+          // otherwise reassign the old content types to the new metadata
+          activity = ContentKindsToLearningActivitiesMap[this.kind];
+        }
+        return activity;
+      },
+    },
+    $trs: {
+      copies: {
```

Comment:
Could you put all these strings in `commonCoreStrings.js` file so they can be reused? 
If you define them in both here, and in `kolibri/plugins/learn/assets/src/views/ContentCard/index.vue`, they will be extracted twice and end up duplicated on Crowdin, which we are trying to avoid 🙂